lib/linguist/languages.yml

```diff
@@ -3372,6 +3372,7 @@ Python:
   color: "#3572A5"
   extensions:
   - ".py"
+  - ".py3"
```

Comment:
This needs to be listed after `".lmi"` instead. The `extensions` list must be alphabetised, with the exception of the language's most common extension (which is listed first).

Otherwise, the tests won't pass and the build will fail.